fellowship-application/index.html

```diff
@@ -99,73 +99,89 @@ <h1>VHA Innovation Ecosystem Fellowship Applications</h1>
                 <section id="application-process" class="application-process text-center padding-y-5 mobile-lg:padding-y-15">
 
                     <div class="grid-col-12">
-                        <h1>Application Process</h1>
-                        <p class="padding-bottom-5 padding-x-6 mobile-lg:padding-x-0">The application period for the 2024 Entrepreneur in Residence and Senior Innovation Fellowships has closed. Application details for 2025 will be provided here when they become available.</p>
+                        <h1>FY 2025 VHA IE Fellowships Application</h1>
+                        <p class="padding-bottom-5 padding-x-6 mobile-lg:padding-x-0">
+                            Applications for the 2025 VHA Innovation Ecosystem Entrepreneur in Residence and Senior 
+                            Innovation Fellow Cohorts will open May 14, 2024. Application details will be provided here when 
+                            they become available.
+                        </p>
                     </div>
 
                     <div id="connections" class="padding-x-0 mobile-lg:padding-x-5">
                         <div class="grid-container">
                             <div class="grid-row grid-gap tablet:flex-justify-center">
                                 <div class="card one">
                                     <div class="card__container">
-                                        <h2 class="card__header h4">Phase 1</h2>
-                                        <p class="card__body">VHA Innovation Ecosystem (VHA IE) Announces program.</p>
+                                        <h2 class="card__header h4">April 1</h2>
+                                        <p class="card__body">
+                                           <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=Ixtf6a-r7kWCHberJRqzv8Qc7YLkpLNJptNkv8H4TKNUOVhXM0o5RUE5UFJNNTk5VU1WQVVJNFVCMi4u">Signup for VHA IE Fellowships Application Reminders</a> 
+                                        </p>
                                     </div>
                                 </div>
                                 <div class="card two">
                                     <div class="card__container">
-                                        <h2 class="card__header h4">Phase 2</h2>
-                                        <p class="card__body">VHA IE hosts an information call for prospective applicants.</p>
+                                        <h2 class="card__header h4">April 23, 25, 30 + May 9</h2>
+                                        <p class="card__body"><span style="font-weight: bold;">Meet the Fellows</span> Information Session. <a href="https://forms.office.com/Pages/ResponsePage.aspx?id=Ixtf6a-r7kWCHberJRqzv8Qc7YLkpLNJptNkv8H4TKNUOVhXM0o5RUE5UFJNNTk5VU1WQVVJNFVCMi4u">Sign up here</a>.
```

Comment:
The use of "inline styles" is not prohibited. The USWDS has text style classes if you want to use a `<span>` tag. If not, the `<strong>` will suffice.

Reply:
https://designsystem.digital.gov/utilities/text-styles/